CMakeLists.txt

```diff
@@ -395,6 +395,12 @@ install(
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/modulefiles/${PROJECT_NAME}
     COMPONENT setup)
 
+install(
+    FILES
+        ${PROJECT_SOURCE_DIR}/LICENSE
+    DESTINATION ${CMAKE_INSTALL_DOCDIR}
+    COMPONENT setup)
+
```

Comment:
I think these 2 lines are also required along with the above changes.

set ( CPACK_RESOURCE_FILE_LICENSE ${PROJECT_SOURCE_DIR}/LICENSE )
set ( CPACK_RPM_PACKAGE_LICENSE "MIT" )

Reply:
Thanks, @nunnikri . 
CPACK_RESOURCE_FILE_LICENSE is already set in `cmake/ConfigCPack.cmake`, but I added CPACK_RPM_PACKAGE_LICENSE.